### PR TITLE
add upcoming releses for cert-manager scraper

### PR DESCRIPTION
**Pull request by seemywingz**
## Test Plan

Ran the scraper locally and verified output 

## Checklist

- [x] If required, I have updated the Plural documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.

### CODE REVIEW

File: static/compatibilities/cert-manager.yaml [hunk 1]
```diff
@@ -3,80 +3,17 @@ git_url: https://github.com/cert-manager/cert-manager
...
-  requirements: []
-  incompatibilities: []
-  chart_version: 1.13.0
 - version: 1.12.0
-  kube: ['1.30', '1.29', '1.28', '1.27', '1.26', '1.25', '1.24', '1.23', '1.22']
+  kube: [ '1.31', '1.30', '1.29', '1.28', '1.27', '1.26', '1.25', '1.24', '1.23', '1.22' ]
   requirements: []
   incompatibilities: []
   chart_version: 1.12.0
-- version: 1.11.0
```
Comment (michaeljguarino) [inline static/compatibilities/cert-manager.yaml:26]: we shouldn't be removing all these previous versions either

**Review by michaeljguarino — CHANGES_REQUESTED** [review]

File: static/compatibilities/cert-manager.yaml (hunk 1 above)
Comment (seemywingz) [reply static/compatibilities/cert-manager.yaml:26]: I removed them while testing locally, didn't mean to commit that

**Comment by michaeljguarino** [comment]
What code change did you do to change that output?

**Comment by seemywingz** [comment]
> What code change did you do to change that output?

I added the "Upcoming Releases" table to be scraped. 


the missing versions was from me testing, I manually removed and added them back to fix the PR

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**